lib/ui/window.dart

```diff
@@ -867,35 +876,44 @@ enum Brightness {
   light,
 }
 
-/// The [SingletonFlutterWindow] representing the main window for applications
-/// where there is only one window, such as applications designed for
-/// single-display mobile devices.
+/// Deprecated. Will be removed in a future version of Flutter.
+///
+/// This property has been deprecated to prepare for Flutter's upcoming support
+/// for multiple views and eventually multiple windows.
```

Comment:
Are Dart globals referred to as properties? Perhaps this should be "This global ..."